docs/android/v2/guides/token-endpoint.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Auth Token Endpoint Guide
+nav: 2.3
+---
+
+## Get an app token using token endpoint
+
+You can use the token endpoint from your 100ms dashboard while building integration with 100ms. This acts as a tool enabling front-end developers to complete the integration without depending on the backend developers to set up a token generation backend service.
+
+**URL format:** `https://prod-in2.100ms.live/hmsapi/<your-100ms-subdomain>.app.100ms.live/api/token`
```

Comment:
why does this not show token endpoint as the placeholder if this guide is about token endpoint?